app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/LocalUnitsFormModal/LocalUnitsForm/index.tsx

```diff
@@ -647,357 +647,359 @@ function LocalUnitsForm(props: Props) {
-                            <BooleanInput
-                                clearable
-                                label={strings.warehousing}
-                                name="is_warehousing"
-                                value={value.health?.is_warehousing}
-                                onChange={onHealthFieldChange}
-                                readOnly={readOnly}
-                                error={getErrorString(
-                                    healthFormError?.is_warehousing,
-                                )}
-                            />
-                            <BooleanInput
-                                clearable
-                                label={strings.coldChain}
-                                name="is_cold_chain"
-                                value={value.health?.is_cold_chain}
-                                onChange={onHealthFieldChange}
-                                readOnly={readOnly}
-                                error={getErrorString(
-                                    healthFormError?.is_cold_chain,
-                                )}
-                            />
-                            <NumberInput
-                                label={strings.ambulanceTypeA}
-                                name="ambulance_type_a"
-                                value={value.health?.ambulance_type_a}
-                                onChange={onHealthFieldChange}
-                                readOnly={readOnly}
-                                error={getErrorString(
-                                    healthFormError?.ambulance_type_a,
-                                )}
-                            />
-                            <NumberInput
-                                label={strings.ambulanceTypeB}
-                                name="ambulance_type_b"
-                                value={value.health?.ambulance_type_b}
-                                onChange={onHealthFieldChange}
-                                readOnly={readOnly}
-                                error={getErrorString(
-                                    healthFormError?.ambulance_type_b,
-                                )}
-                            />
-                            <NumberInput
-                                label={strings.ambulanceTypeC}
-                                name="ambulance_type_c"
-                                value={value.health?.ambulance_type_c}
-                                onChange={onHealthFieldChange}
-                                readOnly={readOnly}
-                                error={getErrorString(
-                                    healthFormError?.ambulance_type_c,
-                                )}
-                            />
-                        </LocalUnitInputSection>
-                        <LocalUnitInputSection
-                            title={strings.servicesTitle}
+                    <>
```

Comment:
Do we need this empty tag?

Reply:
React fragment was used due to multiple container